### PR TITLE
Icons for main categories and sub categories added & Fractal Instabilities icons added

### PR DESCRIPTION
**Pull request by Linkaaaaa**
Issue #79 Add icons for log categories
Issue #87 Determine fractal instabilities

All of the categories now have an icon related to them except the sub categories of the uncategorized section

Some of the bosses don't have a related miniature icon. As replacement i've used the Legendary Insight for bosses in Wing 1-4 and Legendary Divination for Wing 5-7. Deimos doesn't have its own minuature but there is Saul D'Alessio one.
Skorvald, Arkk and Ai have their tonic as icon, Artsariiv uses the Unstable Fractal Essence icon.
The icons are located in the bosses folder within the ArenaNet folder with license disclosure.

The Fractal instabilities are in the instabilities folder, the code implementation for using them is already in place but it's not used yet (determinating the instabilities of a log hasn't been implemented yet).

### CODE REVIEW

File: ArcdpsLogManager/ArcdpsLogManager.csproj [hunk 1]
```diff
@@ -27,15 +58,102 @@
...
+    <Content Include="Images\ArenaNet\strike_icon_32px.png">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
+    <Content Include="Images\ArenaNet\training_area_32px.png">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
+    <Content Include="Images\ArenaNet\uncategorized_32px.png">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
+    <Content Include="Images\ArenaNet\world_vs_world_321px.png">
```
Comment (Sejsel) [inline ArcdpsLogManager/ArcdpsLogManager.csproj:151]: This file seems unused and looks to have a typo in the name, might be an accident?

**Review by Sejsel — CHANGES_REQUESTED** [review]
Looks pretty good to me, just found a typo and a file that might be accidentally unused :)

File: ArcdpsLogManager/ImageProvider.cs [hunk 2]
```diff
@@ -38,82 +39,121 @@ public class ImageProvider
...
+		private Lazy<Image> MAMAIcon { get; } = new Lazy<Image>(Resources.GetMAMAIcon);
+		private Lazy<Image> SiaxTheCorruptedIcon { get; } = new Lazy<Image>(Resources.GetSiaxTheCorruptedIcon);
+		private Lazy<Image> EnsolyssOfTheEndlessTormentIcon { get; } = new Lazy<Image>(Resources.GetEnsolyssOfTheEndlessTormentIcon);
+		private Lazy<Image> SkorvaldIcon { get; } = new Lazy<Image>(Resources.GetSkorvaldIcon);
+		private Lazy<Image> ArtsariivIcon { get; } = new Lazy<Image>(Resources.GetArtsariivIcon);
+		private Lazy<Image> ArkkIcon { get; } = new Lazy<Image>(Resources.GetArkkIcon);
+		private Lazy<Image> AiKeeperOfThePeakIcon { get; } = new Lazy<Image>(Resources.GetAiKeeperOfThePeakIcon);
+		// FESTIVALS
+		private Lazy<Image> FreezieIcon { get; } = new Lazy<Image>(Resources.GetFreezieIcon);
+		// TRAINING AREAA
```
[the remark this answers is not in the document]
Comment (Linkaaaaa) [reply ArcdpsLogManager/ImageProvider.cs]: Fixed in the latest commit

File: ArcdpsLogManager/ArcdpsLogManager.csproj (hunk 1 above)
Comment (Linkaaaaa) [reply ArcdpsLogManager/ArcdpsLogManager.csproj:150]: 321 was a different icon which was too small, removed from the folder

Comment (Sejsel) [reply ArcdpsLogManager/ArcdpsLogManager.csproj:150]: This content include needs to be removed as well.

File: ArcdpsLogManager/ArcdpsLogManager.csproj [hunk 3]
```diff
@@ -15,6 +15,36 @@
...
+    <None Remove="Images\ArenaNet\fractal_instabilities\outflanked_32px.png" />
+    <None Remove="Images\ArenaNet\fractal_instabilities\slippery_slope_32px.png" />
+    <None Remove="Images\ArenaNet\fractal_instabilities\social_awkwardness_32px.png" />
+    <None Remove="Images\ArenaNet\fractal_instabilities\stick_together_32px.png" />
+    <None Remove="Images\ArenaNet\fractal_instabilities\sugar_rush_32px.png" />
+    <None Remove="Images\ArenaNet\fractal_instabilities\toxic_sickness_32px.png" />
+    <None Remove="Images\ArenaNet\fractal_instabilities\toxic_trail_32px.png" />
+    <None Remove="Images\ArenaNet\fractal_instabilities\vengeance_32px.png" />
+    <None Remove="Images\ArenaNet\fractal_instabilities\we_bleed_fire_32px.png" />
+    <None Remove="Images\ArenaNet\freezie_icon_32px.png" />
```
Comment (Sejsel) [inline ArcdpsLogManager/ArcdpsLogManager.csproj:41]: This file doesn't seem to exist, CI fails on it.

File: ArcdpsLogManager/ArcdpsLogManager.csproj [hunk 4]
```diff
@@ -27,15 +57,102 @@
...
+    <Content Include="Images\ArenaNet\fractal_instabilities\toxic_trail_32px.png">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
+    <Content Include="Images\ArenaNet\fractal_instabilities\vengeance_32px.png">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
+    <Content Include="Images\ArenaNet\fractal_instabilities\we_bleed_fire_32px.png">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
+    <Content Include="Images\ArenaNet\freezie_icon_32px.png">
```
Comment (Sejsel) [inline ArcdpsLogManager/ArcdpsLogManager.csproj:129]: This file doesn't seem to exist, CI fails on it.

**Review by Sejsel — CHANGES_REQUESTED** [review]
freezie_icon_32px.png

**Review by Sejsel — APPROVED** [review]

**Pull request merged**